### PR TITLE
Add base64 support

### PR DESCRIPTION
**Pull request by Saptarshi-Bandopadhyay**
Adds support for base64 (image to base64 and vice versa)

### CODE REVIEW

File: src/loadimg/utils.py [hunk 1]
```diff
@@ -58,12 +59,23 @@ def load_img(
         path = os.path.join(secure_temp_dir, "temp_image.png")
         img.save(path)
         return path
+    elif output_type == "base64":
+        img_file = BytesIO()
+        img.save(img_file, format="JPEG")
```
Comment (not-lain) [inline src/loadimg/utils.py]:
```suggestion
        img.save(img_file, format="PNG")
```

**Review by not-lain — COMMENTED** [review]
Switch to PNG to support transparent images

File: src/loadimg/utils.py (hunk 1 above)
Comment (Saptarshi-Bandopadhyay) [reply src/loadimg/utils.py]: Done!

**Comment by not-lain** [comment]
LGTM, thank youuu

**Pull request merged**